config/m4/cuda.m4

```diff
@@ -1 +1,86 @@
-AM_CONDITIONAL([HAVE_CUDA], [true])
+#
+# Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+# See file LICENSE for terms.
+#
+
+#
+# Check for CUDA support
+#
+cuda_happy="no"
+gdrcopy_happy="no"
+
+AC_ARG_WITH([cuda],
+           [AS_HELP_STRING([--with-cuda=(DIR)], [Enable the use of CUDA (default is no).])],
+           [], [with_cuda=no])
+
+AS_IF([test "x$with_cuda" != "xno"],
+        [AS_IF([test ! -z "$with_cuda" -a "x$with_cuda" != "xyes"],
+            [
+            ucx_check_cuda_dir="$with_cuda"
+            ucx_check_cuda_libdir="$with_cuda/lib64 "
```

Comment:
is it ok to assume lib64?
does cuda ship pkg-config?

Reply:
checked for 64 suffix